Sources/Capsule/_Common.swift

```diff
@@ -0,0 +1,294 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2019 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+import func Foundation.ceil
```

Comment:
FYI, this is available in the standard library as `rounded(.up)`.

Reply:
Thank you for mentioning the standard library method, I've adopted it in 8c413e7.